### ISSUE TITLE
Update request: shotcut 21.09.20 → 23.09.29

### ISSUE THREAD

**Issue by futu2** [NixOS/nixpkgs#258744]
- Package name: shotcut
- Latest released version: 23.09.29

- Current version on the unstable channel: 21.09.20
- Current version on the stable/release channel: 21.09.20

- [ ] Checked the [shotcut: 21.09.20 -> 23.07.29](https://github.com/NixOS/nixpkgs/pull/257835)
But it's not the newest version

**Notify maintainers**

@goibhniu @woffs @peti

-----

Note for maintainers: Please tag this issue in your PR.

**Comment by luzpaz** [comment]
shotcut package is showing v23.11.29 

https://github.com/NixOS/nixpkgs/blob/05fdbaa60295296790cddef6c2fe80e5329e06f7/pkgs/applications/video/shotcut/default.nix#L23

**Comment by woffs** [comment]
https://github.com/NixOS/nixpkgs/pull/269444 was merged yesterday

**Comment by luzpaz** [comment]
I see, it hasn't propagated yet.

I guess this ticket can be closed in that case

**Comment by woffs** [comment]
Yes, I think so. The impatient can `nix run nixpkgs/master#shotcut`

**Comment by raspher** [comment]
Current version on the unstable channel: 24.01.13
    Current version on the stable/release channel: 21.09.20
